--- /Users/slynen/catkin_ws/build/opencv2_catkin/opencv2_src-prefix/src/opencv2_src/modules/legacy/src/calibfilter.cpp	2014-04-11 12:15:26.000000000 +0200
+++ /Users/slynen/catkin_ws/build/opencv2_catkin/opencv2_src-prefix/src/opencv2_src/modules/legacy/src/calibfilter_patched.cpp	2014-10-20 01:06:08.000000000 +0200
@@ -95,11 +95,8 @@
 
     Stop();
 
-    if (latestPoints != NULL)
-    {
-        for( i = 0; i < MAX_CAMERAS; i++ )
-            cvFree( latestPoints + i );
-    }
+    for( i = 0; i < MAX_CAMERAS; i++ )
+        cvFree( latestPoints + i );
 
     if( type == CV_CALIB_ETALON_USER || type != etalonType )
     {
@@ -529,7 +526,6 @@
         return;
     }
 
-    if( latestCounts )
     {
         for( i = 0; i < cameraCount; i++ )
         {
